package.json

```diff
@@ -21,7 +21,7 @@
   "main": "./lib/node-pre-gyp.js",
   "dependencies": {
     "detect-libc": "^1.0.2",
-    "mkdirp": "^0.5.1",
```

Comment:
This change is unnecessary `^0.5.1` allows any version lower than 0.6.

In fact, I checked out node-pre-gyp repo, ran `npm install` and got a clean `npm audit report`:

```
node-pre-gyp$ npm audit --registry https://registry.npmjs.org/

                       === npm audit security report ===

found 0 vulnerabilities
 in 338 scanned packages

node-pre-gyp$ npm ls minimist
node-pre-gyp@0.14.0 /c/Users/Eckoh/sites/node-pre-gyp
├─┬ mkdirp@0.5.4
│ └── minimist@1.2.5
├─┬ rc@1.2.8
│ └── minimist@1.2.5  deduped
└─┬ tape@4.13.2
  └── minimist@1.2.5  deduped

npm ls mkdirp
node-pre-gyp@0.14.0 /c/Users/Eckoh/sites/node-pre-gyp
├── mkdirp@0.5.4
├─┬ nock@9.6.1
│ └── mkdirp@0.5.4  deduped
└─┬ tar@4.4.13
  └── mkdirp@0.5.4  deduped


```